### PR TITLE
More UEFI goodies

### PR DESCRIPTION
**Pull request by elicn**
- Fixes and modifications per @liba2k feedback
- Misc. improvements to UEFI

### CODE REVIEW

File: qiling/profiles/uefi.ql [hunk 1]
```diff
@@ -24,15 +22,14 @@ heap_size		= 0x02000000
 [HOB_LIST]
 # EFI_GLOBAL_VARIABLE
 Guid = 7739f24c-93d7-11d4-9a3a-0090273fc14d
-Table = 0x00000000ffffffff
+# the HOB list must end with an etry type of FFFF
```
Comment (liba2k) [inline qiling/profiles/uefi.ql:25]: etry? do you mean entry? Maybe `HobType of FFFF`

**Comment by liba2k** [comment]
Finished reviewing the changes and testing, Thanks @elicn !

File: qiling/profiles/uefi.ql (hunk 1 above)
Comment (elicn) [reply qiling/profiles/uefi.ql:25]: Indeed; this is a typo. Should have been "_The HOB list must end with an entry whose HobType is FFFF_"

**Comment by xwings** [comment]
Thank you sir!

**Pull request merged**